checker/src/test/java/org/checkerframework/checker/test/junit/NullnessAssumeInitializedTest.java

```diff
@@ -0,0 +1,34 @@
+package org.checkerframework.checker.test.junit;
+
+import org.checkerframework.checker.nullness.NullnessChecker;
+import org.checkerframework.framework.test.CheckerFrameworkPerDirectoryTest;
+import org.junit.runners.Parameterized.Parameters;
+
+import java.io.File;
+import java.util.List;
+
+/** JUnit tests for the Nullness checker. */
+public class NullnessAssumeInitializedTest extends CheckerFrameworkPerDirectoryTest {
+
+    /**
+     * Create a NullnessAssumeInitializedTest.
+     *
+     * @param testFiles the files containing test code, which will be type-checked
+     */
+    public NullnessAssumeInitializedTest(List<File> testFiles) {
+        super(
+                testFiles,
+                org.checkerframework.checker.nullness.NullnessChecker.class,
+                "nullness",
+                "-AcheckPurityAnnotations",
+                "-AassumeInitialized",
+                "-Xlint:deprecation",
+                "-Alint=soundArrayCreationNullness,"
+                        + NullnessChecker.LINT_REDUNDANTNULLCOMPARISON);
+    }
+
+    @Parameters
+    public static String[] getTestDirs() {
+        return new String[] {"nullness-assumeinitialized"};
```

Comment:
I think we should run with `-AassumeInitialized` on all `nullness` tests and then move all tests that depend on initialization in a separate directory. Then we can use the tests in `nullness` for both modes, increasing test coverage.